SwiftExpression/SwiftExpression/Regex.swift

```diff
@@ -20,7 +20,7 @@ public struct Regex {
      
      - parameter pattern: The regular expression pattern
      */
-    public init?(pattern: String) {
+    public init?(pattern: String, options: NSRegularExpression.Options = []) {
         do {
             self.regexPattern = try NSRegularExpression(pattern: pattern, options: [])
```

Comment:
You forgot to pass the options parameter into the `NSRegularExpression` init.

Reply:
oh! Thanks for catching that. I am using this locally with the options, but I must have forgot to commit that line for the pull request :|